src/javascript/grid3_javascript_loadclient.md

```diff
@@ -46,7 +46,7 @@ using e2e. That's similar to home servers by matrix
 
 - network: `dev` for devnet, `test` for testnet
 - mnemonics: used for signing the requests.
-- storeSecret: used to encrypt data while storing in backeds
+- storeSecret: used to encrypt data while storing in backend. It's any word that will be used for encrypting/decrypting the keys on threefold key-value store.
```

Comment:
can we also mention if the store secret is left empty, the grid client will use the mnemonic as the store secret?

Reply:
Great idea! Done. Let me know if anything else is needed.